Source/Editor/Windows/SplashScreen.cpp

```diff
@@ -96,45 +87,34 @@ const Char* SplashScreenQuotes[] =
     TEXT("Loading Simulation"),
     TEXT("Get ready for a surprise!"),
     TEXT("Coffee is my fuel"),
-    TEXT("Installing a free copy of Cyberpunk 2077"),
     TEXT("With great power comes great electricity bill"),
     TEXT("Flax was made in the same city as Witcher 3"),
     TEXT("So JavaScript is a scripting version of Java"),
     TEXT("Good things take time.\n~Someone"),
-    TEXT("Get shit done"),
     TEXT("Hold Tight! Loading Flax"),
     TEXT("That's one small step for a man,\none giant leap for mankind"),
     TEXT("Remember to save your work frequently"),
     TEXT("In case of fire:\ngit commit, git push, leave building"),
     TEXT("Keep calm and make games"),
     TEXT("You're breathtaking!!!"),
-    TEXT("Do regular dogs see police dogs & think,\nOh no it's a cop?"),
     TEXT("Dear Santa,\nDefine naughty."),
     TEXT("Blah, blah"),
     TEXT("My PRECIOUS!!!!"),
     TEXT("YOU SHALL NOT PASS!"),
     TEXT("You have my bow.\nAnd my axe!"),
     TEXT("To the bridge of Khazad-dum."),
     TEXT("One ring to rule them all.\nOne ring to find them."),
-    TEXT("Ladies and gentelman, we got him"),
-    TEXT("Cyberpunk of game engines"),
-    TEXT("That's what she said"),
-    TEXT("Compiling Shaders (93,788)"),
```

Comment:
unreal engine refrence

Reply:
Personally I like this one too, I'm mostly going off of the lengthy discussion in the Discord thread.  There was a second PR proposed to add back quotes and I believe there's a replacement for the compiling shaders joke in there.